resqpy/fault.py

```diff
@@ -1614,28 +1614,18 @@ def k_gap_connection_set(grid, skip_inactive = True, feature_name = 'k gap conne
    return kgcs
 
 
-def add_connection_set_and_tmults(model, fault_incl, tmult_dict):
+def add_connection_set_and_tmults(model, fault_incl, tmult_dict = {}):
    """Add a grid connection set to a resqml model, based on a fault include file and a dictionary of fault:tmult pairs.
 
    Grid connection set added to resqml model, with extra_metadata on the fault interpretation containing the MULTFL values
 
    Args:
       model - resqml model object
       fault_incl - fullpath to fault include file or list of fullpaths to fault include files
-      tmult_dict - dictionary of fault name: tmult pairs (must align with faults in include file)
+      tmult_dict - dictionary of fault name: tmult pairs (must align with faults in include file). Optional, if blank values in the fault.include file will be used instead
```

Comment:
For docs to render nicely, Sphinx expects arguments to be listed with colons rather than dashes. Would also suggest a line break:

```suggestion
      tmult_dict: dictionary of fault name: tmult pairs (must align with faults in include file).
          Optional, if blank values in the fault.include file will be used instead
```

Reply:
Also, it could be helpful to document here what a tmult pair is. I'm guessing a pair of transmissibility multipliers?